packages/test/package.json

```diff
@@ -89,10 +89,10 @@
     "url": "https://github.com/coinbase/rest-hooks/issues"
   },
   "dependencies": {
-    "@testing-library/react-hooks": "~5.0.3"
+    "@testing-library/react-hooks": "~7.0.0"
   },
   "peerDependencies": {
-    "@rest-hooks/core": "^1.0.0-beta.1",
+    "@rest-hooks/core": "^1.0.0-beta.1 || ^1.0.0",
```

Comment:
```suggestion
    "@rest-hooks/core": "^1.0.0-0",
```

Reply:
what does -0 mean? are there docs on this?

Reply:
It includes all 1.0.0 prereleases and also 1.0.0 itself.

you can check on semver.npmjs.com if a range works for you.